tools/android/packaging/xbmc/src/org/xbmc/kodi/Splash.java.in

```diff
@@ -544,8 +544,13 @@ public class Splash extends Activity {
   }
 
   protected void startXBMC() {
-    // NB: We only preload lib@APP_NAME_LC@ to be able to get info on missing symbols.
-    //     This is not normally needed
+    // The dynamic linker on droid is a mess.
+    // Depending on version, it might not be able to properly resolve
+    try
+    {
+      System.loadLibrary("smbclient");
+    }
+    catch (UnsatisfiedLinkError e) {}
```

Comment:
ah, understood, thanks. It wasn't immediately obvious to me that this is there to support shared and static libs. Maybe add this to the comment?